docusaurus/docs/Android/03-ui/04-components/05-message-input.md

```diff
@@ -1,10 +1,94 @@
 # Message Input
 
-<!-- TODO: Import whatever makes sense to import from https://getstream.io/chat/docs/android/message_input_view/?language=kotlin -->
+`MessageInputView` is the view used to create a new chat message. It is where the user will type new messages and send then to the chat, normally placed at the bottom of the chat screen:
 
-## Overview
+|Light|Dark| 
+|---|---|
+|![First custom MessageInputView example](../../assets/message_input_light.png)|![First custom MessageInputView example](../../assets/message_input_dark.png)|
 
-<!-- TODO: Brief description and a couple screenshots with default styling. -->
+It supports the following features:
+
+* Emoticons
+* Attachments
+* Slash Commands
+* Typing events
+* Editing messages
+* Threads
+* Mentions
+* Replies
+
+## Handling actions 
```

Comment:
We should have a **Usage** section first where we show how to add this View to a layout and set it up with a ViewModel, as that's what users should do most of the time. After that, we can have **Handling actions** in case they want custom behaviour.

In general, the most basic things should always go first!

Reply:
Changed